src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -44,31 +44,19 @@ const SettingsAccessLinkManagement = ({ enterpriseUUID }) => {
 
   const handleLinkManagementDisabledSuccess = () => {
     refreshLinks();
-    setisLinkManagementEnabled(false);
+    setIsLinkManagementEnabled(false);
     setIsLinkManagementAlertModalOpen(false);
   };
 
   const handleLinkManagementFormSwitchChanged = (e) => {
     const isChecked = e.target.checked;
     if (isChecked) {
-      setisLinkManagementEnabled(e.target.checked);
+      setIsLinkManagementEnabled(e.target.checked);
```

Comment:
tiniest of nits: maybe replace `e.target.checked` with `isChecked` since we already have its value stored in a variable?